rdm/init_files/data/system.yml

```diff
@@ -1,6 +1,11 @@
 # The name of the project
 project_name: PROJECT
 
+# List optional processing filters in order of application
+post_filters: |
```

Comment:
@orwonthe I haven't finished reviewing, but the tests and implementation look great!

I just thought of a few questions, so I wanted to throw them out there:

- Can you imagine a situation where we would only want to add section numbers to certain documents?
- More generally, can you imagine situations where we would want to apply some of these post-processing filters to only certain documents?

Reply:
This can be controlled by surrounding the `auto_section_numbering` tag with appropriately conditioned `{% if ... %}` or by not including the tag at all.

Reply:
Nice, I think this approach is great.  (BTW, in case you didn't notice, this comment was from a while back)